main/index.md

```diff
@@ -28,17 +28,17 @@ features:
   - icon: 🧑‍🎓️
     title: Learn
     details: Learn how Smart Contracts work on Agoric Platform
-    link: /guides/getting-started/
+    link: /what-is-agoric
```

Comment:
the previous icon goes to what-is-agoric. why change this?

Reply:
maybe this one should go to `/guides/zoe/`?

Reply:
Is there an issue to make clear what problem is being fixed?

These links are not wrong according to March 21 discussion where we landed #1018.

Reply:
@dckc currently, the Learn button directs to the Build section in tabs, and the Build button brings to Learn tab (zoe). But Build should be directed to the Build in the page, and Learn should be Learn inside the page. In the PR you tagged, and the Miro link in it does not tell why they are being cross wired.

Reply:
@ansabgillani , dan is right that the Learn should go to `/guides/zoe/` because the subtitle of Learn mentioned Smart Contract that is covered mainly in Zoe page of Learn section.

Reply:
> currently, the Learn button directs to the Build section in tabs, and the Build button brings to Learn tab

ah.

The order is build, learn elsewhere; maybe change it here too?

Reply:
Got it. Resolved.